translate/storage/lisa.py

```diff
@@ -327,15 +328,18 @@ def serialize(self, out=None):
         """Converts to a string containing the file's XML"""
         root = self.document.getroot()
         if self.XMLdoublequotes:
-            out.write(b'<?xml version="1.0" encoding="UTF-8"?>\n')
+            if self.XMLuppercaseEncoding:
+                out.write(b'<?xml version="1.0" encoding="UTF-8"?>\n')
+            else:
+                out.write(b'<?xml version="1.0" encoding="utf-8"?>\n')
         if self.XMLindent:
             reindent(root, **self.XMLindent)
         if 1:
             treestring = etree.tostring(
                 self.document,
                 pretty_print=not self.XMLindent,
                 xml_declaration=not self.XMLdoublequotes,
-                encoding=self.encoding,
+                encoding=(self.encoding.upper() if self.XMLuppercaseEncoding else self.encoding),
```

Comment:
Is this needed? I don't think this has effect on the output.

Reply:
Indeed. Fixed